src/subdomains/supporting/bank/bank-tx/bank-tx.service.ts

```diff
@@ -77,8 +77,6 @@ export class BankTxService {
     // TODO später auskommentieren
     // if (bankTx.type && bankTx.type != BankTxType.UNKNOWN) throw new ConflictException('BankTx Type already set');
 
-    bankTx.type = dto.type;
-
     switch (bankTx.type) {
```

Comment:
I think this is not a good idea, we will create entities (BuyCrypto, BankTxReturn, or BankTxRepeat) multiple times. Bank Tx type should not be changeable IMO 🤔